spec/controllers/signatures_controller_spec.rb

```diff
@@ -5,6 +5,16 @@
     context "signature of user who is not the petition's creator" do
       let(:petition) { FactoryGirl.create(:petition) }
       let(:signature) { FactoryGirl.create(:pending_signature, :petition => petition) }
+      let(:api_url) { "http://data.parliament.uk/membersdataplatform/services/mnis/Constituencies" }
+      let(:fake_body) {
+        "<Constituencies>
```

Comment:
Just wondering if our super cool ConstituencyApi could have some test helper to generate these mock responses for us & maybe help to stub the requests? Just an idea. 

At least we could use ```let(:api_url) { ConstituencyApi::Client.API_URL }```